src/snowflake/snowpark/modin/plugin/_internal/window_utils.py

```diff
@@ -101,3 +100,37 @@ def check_is_rolling_window_supported_by_snowflake(
         ErrorMessage.method_not_implemented_error(
             name="step", class_="Rolling"
         )  # pragma: no cover
+
+
+def check_is_expanding_window_supported_by_snowflake(
+    expanding_kwargs: dict[str, Any]
+) -> None:
+    """
+    Check if execution with snowflake engine is available for the expanding window operation.
+
+    Parameters
+    ----------
+    expanding_kwargs: keyword arguments passed to expanding. The expanding keywords handled in the
+        function contains:
+        min_periods: int, default 1.
+            Minimum number of observations in window required to have a value; otherwise, result is np.nan.
+        axis: int or str, default 0
+            If 0 or 'index', roll across the rows.
+            If 1 or 'columns', roll across the columns.
+            For Series this parameter is unused and defaults to 0.
+        method: str {‘single’, ‘table’}, default ‘single’
+            **This parameter is ignored in Snowpark pandas since the execution engine will always be Snowflake.**
+
+    Returns
+    -------
+    bool
+        Whether operations can be executed with snowflake sql engine.
```

Comment:
this comments seems wrong, this function seems also raises error, and the return value is None here, please update the comment correctly, and update the fucntion to check_and_raise _error if the intention is to also raise an error

Reply:
Good catch, will update.